modules/claims_api/app/workers/claims_api/claim_uploader.rb

```diff
@@ -7,6 +7,8 @@ module ClaimsApi
   class ClaimUploader
     include Sidekiq::Worker
 
+    sidekiq_options 'retry': true, unique_until: :success
```

Comment:
I'm not sure if this is what we want here, but do we want some sort of retry logic if these are intermittently failing (at least on sandbox)?

Reply:
double checking that this is still something we want to add if we have the have the `evss_id` logic added below. I'm not sure how this retry logic differ from any default states there might already be